$NetBSD: patch-src_Makevars,v 1.2 2026/07/04 23:26:00 mef Exp $

symbol backtrace_symbols is defined in /usr/lib/libexecinfo

---------------------------------
** testing if installed package can be loaded from temporary location
Error: package or namespace load failed for 'plyr' in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/tmp/math/R-plyr/work/.destdir/usr/pkg/lib/R/library/00LOCK-plyr/00new/plyr/libs/plyr.so':
  /tmp/math/R-plyr/work/.destdir/usr/pkg/lib/R/library/00LOCK-plyr/00new/plyr/libs/plyr.so: Undefined PLT symbol "backtrace_symbols" (symnum = 111)
Error: loading failed
Execution halted
ERROR: loading failed
---------------------------------

--- src/Makevars.orig	2026-07-05 08:20:00.751584080 +0900
+++ src/Makevars	2026-07-05 08:22:56.091779410 +0900
@@ -0,0 +1,5 @@
+ #### Debugging flags ####
+ # Uncomment to enable thread assertions
+ # PKG_CPPFLAGS += -DDEBUG_THREAD -UNDEBUG
+
+@@PKG_LIBS@@
